composer.json

```diff
@@ -103,7 +103,7 @@
   },
   "scripts": {
     "auto-scripts": {
-
+      "requirements-checker": "script"
```

Comment:
We've removed all these scripts because they hose our docker builds when they run on campus install. Instead this should go in CI as a separate step.

Reply:
yanked it. :heavy_check_mark: